sources/Outputs/src/Par.cpp

```diff
@@ -81,7 +81,19 @@ Par::updateSignalNGenerator(boost::shared_ptr<parameters::ParametersSet> set) {
   set->addParameter(helper::buildParameter("generator_QMax", constants::powerValueMax));
   set->addParameter(helper::buildParameter("generator_PMin", -constants::powerValueMax));
   set->addParameter(helper::buildParameter("generator_PMax", constants::powerValueMax));
-  set->addReference(helper::buildReference("generator_PNom", "p_pu", "DOUBLE"));
+
+  switch (def_.activePowerCompensation) {
+  case dfl::inputs::Configuration::ActivePowerCompensation::P:
+    set->addReference(helper::buildReference("generator_PNom", "p_pu", "DOUBLE"));
+    break;
+  case dfl::inputs::Configuration::ActivePowerCompensation::TARGET_P:
+    set->addReference(helper::buildReference("generator_PNom", "targetP_pu", "DOUBLE"));
+    break;
+  case dfl::inputs::Configuration::ActivePowerCompensation::PMAX:
```

Comment:
Add a comment or do nothing case for default, to explain that ther is no default case by definition of the enum.

Reply:
Do you want me to factorize the case ?

Reply:
No, juste the info for the default case. It is a good practice to help maintenability and automatic verification by tools